build/common.sh

```diff
@@ -770,6 +770,7 @@ function kube::release::package_full_tarball() {
   cp -R "${KUBE_ROOT}/third_party/htpasswd" "${release_stage}/third_party/htpasswd"
 
   cp -R "${KUBE_ROOT}/examples" "${release_stage}/"
+  cp -R "${KUBE_ROOT}/docs" "${release_stage}/"
```

Comment:
also required for e2e tests which succeeded locally after the move but failed on jenkins which only has the tarball, not the full source.